docs/operator-manual/reconcile.md

```diff
@@ -111,3 +112,55 @@ data:
     # actually changing in content.
     - .status.conditions[].lastTransitionTime
 ```
+
+## Ignoring updates for untracked resources
+
+ArgoCD will only apply `ignoreResourceUpdates` configuration to tracked resources of an application. This means dependant resources, such as a `ReplicaSet` and `Pod` created by a `Deployment`, will not ignore any updates and trigger a reconcile of the application for any changes.
+
+If you want to apply the `ignoreResourceUpdates` configuration to an untracked resource, you can add the
+`argocd.argoproj.io/ignore-resource-updates=true` annotation in the dependent resources manifest.
```

Comment:
```suggestion
Argo CD will only apply `ignoreResourceUpdates` configuration to tracked resources of an application. This means child resources, such as a `ReplicaSet` and `Pod` created by a `Deployment`, will not ignore any updates and trigger a reconcile of the application for any changes.

If you want to apply the `ignoreResourceUpdates` configuration to a child resource, you can add the
`argocd.argoproj.io/ignore-child-resource-updates=true` annotation in the dependent resources manifest.
```

Reply:
@crenshaw-dev the annotation is to be set on the Actual resource, so for a Pod created by a CronJob, the annotation would be added in the PodSpec to ignore "self" updates. IMO, I think `ignore-child-resource-updates` is confusing. Furthermore, this can be applied to any resources watched by argo, not only child resources. 

Based on https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/, dependent seems to be the proper terminology.

```suggestion
ArgoCD will only apply `ignoreResourceUpdates` configuration to tracked resources of an application. This means [dependent resources](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/), such as a `ReplicaSet` and `Pod` created by a `Deployment`, will not ignore any updates and trigger a reconcile of the application for any changes.

If you want to apply the `ignoreResourceUpdates` configuration to an untracked resource, you can add the
`argocd.argoproj.io/ignore-resource-updates=true` annotation in the dependent resources manifest.
```

Reply:
Ah! I skimmed too quickly and misunderstood the placement of the annotation. Thanks for clarifying!